mautic/mautic
# Updates leads primary company name after company name changed

…mary

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5652
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Required: )
#### Description:
Contact's primary company name wasn't change after company edit.
Reported here https://github.com/mautic/mautic/issues/5652

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Assign company to contact 
2.  Change company name
3. See contacts detail  and see old company name  (or see column company in leads table)

#### Steps to test this PR:
1.  Repeat all steps
2.  See company name was update
